.github/workflows/main.yml

```diff
@@ -31,6 +31,9 @@ jobs:
       - name: Restrict Symfony version
         if: matrix.symfony
         run: rm composer.lock && composer require "symfony/symfony:${{ matrix.symfony }}" --no-update
+      - name: Force Monolog
+        if: matrix.symfony == '^6.4'
+        run: composer require "monolog/monolog" --no-update
```

Comment:
Why do we need this for 6.4 only?

Reply:
It's weird, but it fails otherwise. Look at the Actions, the last failing one. It doesn't find a monolog class, and I don't understand why.  If you have a better solution, let me know.

Reply:
Due to the force push, I cannot see it...

Reply:
> Due to the force push, I cannot see it...

it's still visible here https://github.com/facile-it/crossbar-http-publisher-bundle/actions/runs/9495808250/job/26168834801

Reply:
It seems that it's trying to load the Symfony Monolog Bridge's DebugProcessor, which is activated here: https://github.com/symfony/symfony/blob/7b5377056d151024d994757aba12db556aa2ee28/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1180-L1188

As you can see, it can be shut off by disabling debug mode, which should also be the correct way to proceed in a test; try `debug: false` in the app that you're spinning up during the tests and that should be fixed. It's strange that it gets activated though, since I see a `class_exists` check too...

Reply:
That's ugly. Probably this is caused by the committed `composer.lock`. We should try to remove it in CI before other dependencies are installed.

Btw I think it's useless to even have a committed composer.lock in a library project that aims to provide such a wide compatibility. What about it?

Reply:
I thought the same at first, but then I realized that in the Actions we perform a composer update. So it's not a lock-related problem.
Anyway, I agree that keeping the lock doesn't make much sense.

Reply:
If you have a `composer.lock`, `composer update` can still have a different behavior; I agree with @ilario-pierbattista, we should aim to remove it.